### ISSUE TITLE
Add who-is-who of modders

### ISSUE THREAD

**Issue by blizzy78** [flashcactus/kountdown#2]
What would be nice to have is a who-is-who of modders, so that you can ask Kountdown who-the-EFF someone is:

```
me> .modder ferram4
Kountdown> ferram4 is responsible for FAR, NEAR, KIDS, KJR.
```

**Comment by Lokosmos** [comment]
Yes, this is great. You can make it changeable via github, so we can populate it for you.

**Comment by flashcactus** [comment]
@Lokosmos I'm not sure I want to add any kind of github sync to the bot. Simply publishing the info file would probably be okay though

**Comment by janbrohl** [comment]
You could get info about who made which mods using [CKAN mod-index](https://github.com/KSP-CKAN/CKAN-meta) but as many mods are not in that index this would not be totally reliable.
